docs/rtd_requirements.txt

```diff
@@ -1,4 +1,5 @@
--i https://pypi.python.org/simple
+-i http://localhost:3141/root/pypi/+simple/
+--trusted-host localhost:3141
```

Comment:
What is this?

Reply:
t's because I have a local devpi instance deployed.

AFAIK, it's not a fatal issue. It just says where those things were pulled from.

Reply:
Let's change this back. When I run it locally, i get:
```
╰─$ pip install -r docs/rtd_requirements.txt 
adding trusted host: 'localhost:3141' (from line 2 of docs/rtd_requirements.txt)
Looking in indexes: http://localhost:3141/root/pypi/+simple/
```
which fails to find anything.

Reply:
ok. i'll fix that. :slightly_smiling_face:

Reply:
fixed!